docs/security/shiroauthentication.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Did you add this new line on purpose? If not, you need to remove this empty line. Since Zeppelin docs website is using Jekyll as a site builder, it needs to recognize `___`(triple-dash lines).